apps/server/src/routes/login.ts

```diff
@@ -114,6 +116,7 @@ function login(req: Request, res: Response) {
 
     const submittedPassword = req.body.password;
     const submittedTotpToken = req.body.totpToken;
+    const submittedUsername = req.body.username; // New field for multi-user mode
```

Comment:
There seems to be server-side support for user management, but I don't see any way to create a new user or to login, as user.

Without the front-end functionality to back it up, it would be non-functional.